include/auxiliary.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
+#include <basics.h>
+#include <config.h>
+#include <seqan/bam_io.h>
+#include <seqan/seq_io.h>
+#include <iostream>
+#include <memory>
+#include <thread>
+
+struct Auxiliary{
+    std::ofstream                                  log_file;
```

Comment:
remove the spaces between the type and name

Reply:
Hmmm you mean don't align the names of all the variables?

Reply:
yes. I think I never did that in the other structs, did I?

Reply:
No I think I did it out of habit...